### PR TITLE
add angular, nextjs, react, vue to all page meta and fragments that currently show javascript

### PR DESCRIPTION
**Pull request by katiegoines**
#### Description of changes:

JS content should appear for angular/nextjs/react/vue pages as well as javascript pages. These platforms have been added to the appropriate pages meta.platforms and to the fragments.

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by hbuchel — COMMENTED** [review]
Can you do a little package.json/yarn.lock cleanup? (sorry that is probably annoying since you need them to do the migration but that will all be over soooooooon)

File: src/pages/[platform]/build-a-backend/datastore/customize-primary-keys/index.mdx [hunk 1]
```diff
@@ -34,4 +34,4 @@ import flutter0 from '/src/fragments/lib/datastore/native_common/advanced-workfl
 
 import js0 from '/src/fragments/lib/datastore/native_common/advanced-workflows.mdx';
 
-<Fragments fragments={{ javascript: js0, 'react-native': js0 }} />
+<Fragments fragments={{ javascript: js0, angular: js0, nextjs: js0, react: js0, vue: js0 }} />
```
Comment (jacoblogan) [inline src/pages/[platform]/build-a-backend/datastore/customize-primary-keys/index.mdx:37]: Looks like we are losing `react-native` on this fragment, was that intentional? if so do we also need to remove it from the `platforms` list above?

File: src/pages/[platform]/build-a-backend/graphqlapi/upgrade-guide/index.mdx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export function getStaticProps(context) {
 
 import js0 from '/src/fragments/lib/graphqlapi/js/upgrade-guide.mdx';
 
-<Fragments fragments={{ javascript: js0, 'react-native': js0 }} />
+<Fragments fragments={{ javascript: js0, angular: js0, nextjs: js0, react: js0, vue: js0 }} />
```
Comment (jacoblogan) [inline src/pages/[platform]/build-a-backend/graphqlapi/upgrade-guide/index.mdx:25]: again losing `react-native` here (ignore if that was intentional)

File: src/pages/[platform]/prev/build-a-backend/auth/remember-device/index.mdx [hunk 3]
```diff
@@ -90,23 +100,47 @@ You can mark your device as remembered:
 
 import js2 from '/src/fragments/lib-v1/auth/js/device_features/10_rememberDevice.mdx';
 
-<Fragments fragments={{ javascript: js2, 'react-native': js2 }} />
+<Fragments
```
Comment (jacoblogan) [inline src/pages/[platform]/prev/build-a-backend/auth/remember-device/index.mdx:103]: missing `react-native`

File: src/pages/[platform]/prev/build-a-backend/auth/remember-device/index.mdx [hunk 4]
```diff
@@ -90,23 +100,47 @@ You can mark your device as remembered:
...
 ### Forget Device
 
 You can forget your device by using the following API. Note that forgotten devices are still tracked. See below for the difference between remembered, forgotten and tracked.
 
 import js6 from '/src/fragments/lib-v1/auth/js/device_features/20_forgetDevice.mdx';
 
-<Fragments fragments={{ javascript: js6, 'react-native': js6 }} />
+<Fragments
+  fragments={{
+    javascript: js6,
```
Comment (jacoblogan) [inline src/pages/[platform]/prev/build-a-backend/auth/remember-device/index.mdx:121]: missing `react-native`

File: src/pages/[platform]/prev/build-a-backend/auth/remember-device/index.mdx [hunk 5]
```diff
@@ -90,23 +100,47 @@ You can mark your device as remembered:
...
 ### Fetch Devices
 
 You can fetch a list of remembered devices by using the following:
 
 import js10 from '/src/fragments/lib-v1/auth/js/device_features/30_fetchDevice.mdx';
 
-<Fragments fragments={{ javascript: js10, 'react-native': js10 }} />
+<Fragments
+  fragments={{
+    javascript: js10,
```
Comment (jacoblogan) [inline src/pages/[platform]/prev/build-a-backend/auth/remember-device/index.mdx:137]: missing `react-native`

File: src/pages/[platform]/prev/build-a-backend/datastore/customize-primary-keys/index.mdx [hunk 6]
```diff
@@ -30,4 +30,4 @@ import flutter0 from '/src/fragments/lib-v1/datastore/native_common/advanced-wor
 
 import js0 from '/src/fragments/lib-v1/datastore/native_common/advanced-workflows.mdx';
 
-<Fragments fragments={{ javascript: js0, 'react-native': js0 }} />
+<Fragments fragments={{ javascript: js0, angular: js0, nextjs: js0, react: js0, vue: js0 }} />
```
Comment (jacoblogan) [inline src/pages/[platform]/prev/build-a-backend/datastore/customize-primary-keys/index.mdx:33]: missing `react-native`

File: src/pages/[platform]/prev/build-a-backend/storage/transfer-acceleration/index.mdx [hunk 7]
```diff
@@ -22,4 +22,4 @@ export function getStaticProps(context) {
 
 import js from '/src/fragments/lib-v1/storage/js/transfer-acceleration.mdx';
 
-<Fragments fragments={{ javascript: js, 'react-native': js }} />
+<Fragments fragments={{ javascript: js, angular: js, nextjs: js, react: js, vue: js }} />
```
Comment (jacoblogan) [inline src/pages/[platform]/prev/build-a-backend/storage/transfer-acceleration/index.mdx:25]: missing `react-native`

**Review by jacoblogan — APPROVED** [review]

**Review by timngyn — APPROVED** [review]

**Pull request merged**